.changeset/thick-apes-build.md

```diff
@@ -0,0 +1,5 @@
+---
+"@example/blog": patch
```

Comment:
can you remove this file? The examples don't get published so this file is not needed.